cv32e40x/tests/cfg/default.yaml

```diff
@@ -2,8 +2,7 @@ name: default
 description: Default configuration for CV32E40X simulations
 compile_flags:
 ovpsim: >
-    --override root/cpu/misa_Extensions=0x001104
```

Comment:
Why are these OVPsim overrides no longer necessary?

Reply:
Ah.  I had forgotten about this.   A far better approach IMHO (as you know I like to keep as much cfg info as possible under direct control of the UVM testcase and this is a step in that direction).